Cargo.toml

```diff
@@ -9,7 +9,7 @@ members = [
   "kairos-test-utils",
   "kairos-tx",
   "kairos-prover/kairos-circuit-logic",
-  "demo-contract-tests",
+  "demo-contract-tests", "kairos-data",
```

Comment:
Nit: add newline here

Reply:
No idea why I left it like that, will fix